workflows/Integration_of_InsightVM_and_Deep_Security_Virtual_Patching/workflow.spec.yaml

```
@@ -0,0 +1,33 @@
+extension: workflow
+products: ["insightconnect"]
+name: Integration_of_InsightVM_and_Deep_Security_Virtual_Patching
+title: "Integration of InsightVM and Deep Security Virtual Patching"
+description: Automatically protect your assets against exploitation of vulnerabilities found by InsightVM with Deep Security virtual patching.
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [vulnerability_management, remediation_management]
+  keywords: [vulnerability_management, remediation_management, insightvm, trend_micro, deep_security, virtual_patching, intrusion_prevention]
```

We don't want to duplicate keywords with use_cases

```suggestion
  keywords: [insightvm, trend_micro, deep_security, virtual_patching, intrusion_prevention]
```